node-template/runtime/src/lib.rs

```diff
@@ -168,6 +168,8 @@ impl timestamp::Trait for Runtime {
 impl balances::Trait for Runtime {
 	/// The type for recording an account's balance.
 	type Balance = u128;
+	/// To convert transaction weight to fee
```

Comment:
I'd leave that as a TODO (probably do it myself) to actually fill this and document the whole weight fee process in `node-template` as well.